src/routes/MapExplorer/MapDrawer/MapDrawer.js

```diff
@@ -41,6 +41,7 @@ function MapDrawer (props) {
       querystringParams={querystringParams}
       post={post}
       styleName='contentCard'
+      constrained={'true'}
```

Comment:
you want this to just be {true} not {'true'} though it will work with the string because that evaluates to true

Reply:
Linter gave me ann error withh {true}, after googling I put in {'true'} and linter stopped complaining